### PR TITLE
remove outdated repos

### PR DESCRIPTION
**Pull request by thefabbulus**
Closes https://github.com/markets/awesome-ruby/issues/988

**ARCHIVED BY OWNER / UNMANTAINED / MERGED**
* celluloid
* celluloid/reel
* stuff-classifier
* ratom
* legacy-linuxbrew
* dm-core
* abracadabra
* mailman
* doctor
* payola
* blender
* deploynator
* kochiku
* datamapper do
* authority
* facy
* rack-protection
* rails-footnotes
* time-lord
* rails-api

**UNMANTAINED / LAST COMMIT <= 2017**
* goliath 25 issues 3 open PR
* render_sync 26 issues 13 open PR
* websocket-rails 171 issues 33 open PR
* anemone 21 open PR
* volt 58 issues 18 open PR
* machinist 18 issues 15 open PR
* octopress 45 issues 5 open PR
* konacha 18 issues 4 open PR
* spork 101 issues 13 open PR
* yomu 19 issues 7 open PR
* searchlogic 115 issues 1 open PR
* rb-gsl 21 issues 7 open PR
* algorithms 11 issues 9 open PR
* perftools.rb 25 issues 3 open PR
* ruby-traceview 4 issues 2 open PR
* god 89 issues 26 open PR
* gimli 19 issues 4 open PR
* shrimp 11 issues 9 open PR
* wisepdf 5 issues 10 open PR
* mongomapper 57 issues 23 open PR
* perpetuity 12 issues open PR
* espinita 4 issues 12 open PR
* active_importer 10 issues open PR
* mini_record 10 issues 6 open PR
* acts_as_commentable 9 issues 12 open PR
* acts_as_commentable_with_threading 17 issues 5 open PR
* acts_as_followerm 36 issues 20 open PR
* activerecord-reputation-system 26 issues 2
* acts-as-taggable-on 229 issues 36 open PR
* acts_as_votable 20 issues 5 open PR
* play 23 issues 7 open PR
* hirb 12 issues 5 open PR
* ruby-cabin 12 issues 7 open PR
* eeepub 9 issues 9 open PR
* git-scribe 20 issues 3 open PR
* json-compare 4 issues 10 open PR
* koudoku 55 issues 15 open PR
* request-log-analyzer 16 issues 5 open PR
* barkeep 167 issues 18 open PR
* less.rb 12 issues 5 open PR
* declarative_authorization 50 issues 15 open PR
* apotomo 25 issues 7 open PR
* typus 47 issues 15 open PR
* analytical 5 issues 6 open PR
* gabba 8 issues 1 open PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -353,7 +344,6 @@ Thanks to all [contributors](https://github.com/markets/awesome-ruby/graphs/cont
 
 ## Concurrency and Parallelism
 
-* [Celluloid](http://celluloid.io) - Actor-based concurrent object framework for Ruby.
```
Comment (markets) [inline README.md:356]: sad for this, but hopefully will have something similar in Ruby 3 core (Guilds/Ractor): https://www.reddit.com/r/ruby/comments/gbhkjn/guild_renamed_to_ractor/

File: README.md [hunk 2]
```diff
@@ -526,14 +509,12 @@ Best suited for map-reduce or e.g. parallel downloads/uploads.
 
 * [Diffy](https://github.com/samg/diffy) - Easy Diffing With Ruby.
 * [gemdiff](https://github.com/teeparham/gemdiff) - Find source repositories for gems. Open, compare, and update outdated gem versions.
-* [JsonCompare](https://github.com/a2design-inc/json-compare) - Returns the difference between two JSON files.
```
Comment (markets) [inline README.md:529]: this is probably still working in newer Rubies, I'll keep it for now.

File: README.md [hunk 3]
```diff
@@ -551,8 +532,6 @@ Best suited for map-reduce or e.g. parallel downloads/uploads.
 * [Braintree](https://github.com/braintree/braintree_ruby) - Braintree Ruby bindings.
 * [Conekta](https://github.com/conekta/conekta-ruby) - Conekta Ruby bindings.
 * [credit_card_validations](https://github.com/Fivell/credit_card_validations) - A ruby gem for validating credit card numbers, generating valid numbers, Luhn checks.
-* [Koudoku](https://github.com/andrewculver/koudoku) - Robust subscription support for Ruby on Rails apps using Stripe, including out-of-the-box pricing pages, payment pages, and subscription management.
-* [Payola](https://github.com/peterkeen/payola) - Drop-in Rails engine for accepting payments with Stripe.
```
Comment (markets) [inline README.md:555]: very sad 😢 this was really cool

File: README.md [hunk 4]
```diff
@@ -821,7 +794,6 @@ Best suited for map-reduce or e.g. parallel downloads/uploads.
 
 * [Coltrane](https://github.com/pedrozath/coltrane) - A music theory library with a command-line interface.
 * [Maestro](https://github.com/smashingboxes/maestro) - A Slack-Powered music bot for Spotify
-* [play ►](https://github.com/play/play) - Your company's dj.
```
Comment (markets) [inline README.md:824]: 😢 this was also really cool

File: README.md [hunk 5]
```diff
@@ -935,7 +895,6 @@ Best suited for map-reduce or e.g. parallel downloads/uploads.
   * [Berkshelf](https://github.com/berkshelf/berkshelf) - A Chef Cookbook manager.
   * [CocoaPods](https://github.com/CocoaPods/CocoaPods) - The Objective-C dependency manager.
   * [fpm](https://github.com/jordansissel/fpm) - Effing package management! Build packages for multiple platforms (deb, rpm, etc) with great ease and sanity.
-  * [Linuxbrew](https://github.com/Homebrew/linuxbrew) - A fork of Homebrew for Linux.
```
Comment (markets) [inline README.md:938]: This moved to another repo, instead of ✂️ we should update to new place

File: README.md [hunk 6]
```diff
@@ -979,7 +934,6 @@ Best suited for map-reduce or e.g. parallel downloads/uploads.
 * [Bluepill](https://github.com/bluepill-rb/bluepill) - Simple process monitoring tool.
 * [Eye](https://github.com/kostya/eye) - Process monitoring tool. Inspired from Bluepill and God.
 * [Foreman](https://github.com/ddollar/foreman) - Manage Procfile-based applications.
-* [God](https://github.com/mojombo/god) - An easy to configure, easy to extend monitoring framework written in Ruby.
```
Comment (markets) [inline README.md:982]: maybe we should keep this one: https://github.com/mojombo/god/issues/256

File: README.md [hunk 7]
```diff
@@ -1119,7 +1068,6 @@ Best suited for map-reduce or e.g. parallel downloads/uploads.
 * [bundler-audit](https://github.com/rubysec/bundler-audit) - Patch-level security verification for Bundler.
 * [Metasploit](https://github.com/rapid7/metasploit-framework) - World's most used penetration testing software.
 * [Rack::Attack](https://github.com/kickstarter/rack-attack) - Rack middleware for blocking & throttling abusive requests.
-* [Rack::Protection](https://github.com/sinatra/rack-protection) - Rack middleware for protecting against typical Web attacks.
```
Comment (markets) [inline README.md:1122]: this moved to https://github.com/sinatra/sinatra/tree/master/rack-protection

**Comment by markets** [comment]
Such a clean up @thefabbulus thanks 👍 I added some comments that needs to be addressed before merging (and the merge conflict 🙏).

**Comment by thefabbulus** [comment]
should be ready @markets

**Comment by markets** [comment]
Thanks :rocket:

**Pull request merged**